### ISSUE TITLE
imported less files

### ISSUE THREAD

**Issue by trickydisco78** [fuse-box/fuse-box#279]
I'm having problems bundling less files located in a sub folder.

I have less.less which imports 1 file in semantic-ui/test.less

I'm getting this output
![capture](https://cloud.githubusercontent.com/assets/3627429/23209898/1c28730a-f8f3-11e6-9a28-7c66d50006fc.PNG)

Cannot find semantic-ui/test.less wasn't found tried - semantic-ui/test.less,semantic-ui\test.less
{ [Error: 'semantic-ui/test.less' wasn't found. Tried - semantic-ui/test.less,semantic-ui\test.less]

**Comment by trickydisco78** [comment]
So it appears i need to put the root relative path to the less files

so "src/semantic-ui/test.less" instead of file relative "semantic-ui/test.less"

This behaviour is different to importing sass that i've tried which does file relative

so "sass/sass.scss works"

**Comment by nchanged** [comment]
Hm..so what would be a fix you suggest? I am a bit uncertain...

**Comment by trickydisco78** [comment]
To have the same behaviour as importing any sass/less file. To go file relative. I shouldn't need to fully qualify the file from the project root.

**Comment by NGPixel** [comment]
The fix is simply to specify the paths in the LESSPlugin options:

```js
fsbx.LESSPlugin({
   paths: ['./src'],
})
```

Maybe this could be set by default to the value of `homeDir` which is what devs would expect?

**Comment by nchanged** [comment]
That's a good idea! @NGPixel

**Comment by guillaume86** [comment]
I added this in LESSPlugin.js to fix relative imports:

```
        if (!options.path) {
            options.path = [];
        }
        options.path.push(file.info.absDir);`
```

Not sure if it should be default (could break a use case?).

**Comment by nchanged** [comment]
How about this http://fuse-box.org:3333/plugins/sass-plugin#macros
?

**Comment by guillaume86** [comment]
It seems to be for different use cases (overkill for my case), I'm just doing this:

file1.less contains:
`@import "./file2.less";`

and file2.less is in the same directory.

**Comment by devmondo** [comment]
this is supposed to be fixed in version 2.0 and the new update to the LessPlugin.

we will close this for now, but feel free to open it otherwise :)
